rules/integrations/aws/credential_access_new_terms_secretsmanager_getsecretvalue.toml

```diff
@@ -4,14 +4,18 @@ integration = ["aws"]
 maturity = "production"
 min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
 min_stack_version = "8.9.0"
-updated_date = "2024/04/14"
+updated_date = "2024/05/06"
 
 [rule]
 author = ["Nick Jones", "Elastic"]
 description = """
-An adversary with access to a compromised AWS service such as an EC2 instance, Lambda function, or other service may attempt to leverage the compromised service to access secrets in AWS Secrets Manager. This rule looks for the first time a specific user identity has programmatically retrieved a specific secret value from Secrets Manager using the `GetSecretValue` action.
-
-This rule assumes that AWS services such as Lambda functions and EC2 instances are setup with IAM role's assigned that have the necessary permissions to access the secrets in Secrets Manager. An adversary with access to a compromised AWS service such as an EC2 instance, Lambda function, or other service would rely on the compromised service's IAM role to access the secrets in Secrets Manager.
+An adversary with access to a compromised AWS service such as an EC2 instance, Lambda function, or other service may
+attempt to leverage the compromised service to access secrets in AWS Secrets Manager. This rule looks for the first time
+a specific user identity has programmatically retrieved a specific secret value from Secrets Manager using the
```

Comment:
```suggestion
a specific user identity has programmatically retrieved a secret value from Secrets Manager using the
```

Reply:
Also need the same edit in the investigation guide along with changing "within the last 15 days" to 
"10 days"